src/commands/workspace/stop.ts

```diff
@@ -10,22 +10,21 @@
 
 import { Command, flags } from '@oclif/command'
 import { cli } from 'cli-ux'
-import Listr = require('listr')
 import * as notifier from 'node-notifier'
 
-import { accessToken, cheNamespace, skipKubeHealthzCheck } from '../../common-flags'
-import { CheTasks } from '../../tasks/che'
-import { ApiTasks } from '../../tasks/platforms/api'
-import { WorkspaceTasks } from '../../tasks/workspace-tasks'
+import { CheHelper } from '../../api/che'
+import { CheApiClient } from '../../api/che-api-client'
+import { KubeHelper } from '../../api/kube'
+import { accessToken, ACCESS_TOKEN_KEY, cheApiUrl, cheNamespace, CHE_API_URL_KEY } from '../../common-flags'
 
 export default class Stop extends Command {
   static description = 'Stop a running workspace'
 
   static flags = {
     help: flags.help({ char: 'h' }),
-    'access-token': accessToken,
+    [CHE_API_URL_KEY]: cheApiUrl,
+    [ACCESS_TOKEN_KEY]: accessToken,
     chenamespace: cheNamespace,
-    'skip-kubernetes-health-check': skipKubeHealthzCheck
```

Comment:
Pls, don't remove this flag even it isn't used. 
The commands the use it will fail. 
Restore it in all commands.

Reply:
I think this flag is used really rarely... @tolusha do you insist on restoring it with deprecation message?

Reply:
No, just keep the flag.